src/sharepoint/task/TASK.html

```diff
@@ -34,7 +34,7 @@
             </tbody>
 
             <tbody data-ng-if="!taskCtrl.loaders.tasks && taskCtrl.tasksIds.length > 0">
-                <tr data-ng-repeat="task in taskCtrl.taskDetails track by $index">
+                <tr data-ng-repeat="task in taskCtrl.taskDetails | orderBy:'-todoDate' track by $index">
```

Comment:
What's the format of the `todoDate` attribute ? Because this would work only if the format is `YYYY-MM-DD`